dist/vl-http-error-message.js

```diff
@@ -205,3 +209,24 @@ define('vl-http-404-message', class extends VlHttpErrorMessage {
         });
     }
 });
+
+define('vl-http-500-message', class extends VlHttpErrorMessage {
+    constructor() {
+        super({
+            title: 'Oeps, er is iets fout gelopen',
+            image: 'https://cdn.milieuinfo.be/http-error-message-assets/LATEST/img/unexpected-error.svg',
+            imageAlt: 'Onverwachte fout',
+            text: `<p>De actie die u heeft uitgevoerd is fout gelopen. Gelieve opnieuw te proberen.</p>`,
```

Comment:
moet hier niks komen van "gelieve de helpdesk te contacteren als het probleem zich blijft voordoen"?

Reply:
De tekst kan altijd gewijzigd worden door afnemers, maar misschien een functionele vraag nu voor @kevcraey om de standaard tekst te kiezen.

Ter info @kevcraey ik had nu gekozen voor:
- Oeps, er is iets fout gelopen
- De actie die u heeft uitgevoerd is fout gelopen. Gelieve opnieuw te proberen.

Ik kan dit aanpassen naar:
- Oeps, er is iets fout gelopen
- De actie die u heeft uitgevoerd is fout gelopen. Gelieve opnieuw te proberen of contact op te nemen met de helpdesk als het probleem zich blijft voordoen.

Probleem daarbij is, dat het onduidelijk kan zijn voor de gebruiker wie/wat de helpdesk is en hoe daarmee contact opgenomen moet worden. Ik zou een mailto kunnen implementeren op het woord helpdesk, maar welk e-mailadres hanteren we dan?

Reply:
Als ik het goed begrijp is dit default tekst en kunnen afnemers die naar hun eigen hand zetten. Ik zie geen reden om de default dan meer de te laten bevatten dan dit.

Reply:
ca va